pkg/util/constant.go

```diff
@@ -37,6 +37,8 @@ const (
 	LocalHostpathCasType = "localpv-hostpath"
 	// LocalDeviceCasType cas type name
 	LocalDeviceCasType = "localpv-device"
+	// Local host path cas type name
+	LocalHostpath = "local-hostpath"
```

Comment:
I'll confirm about this in an hour for sure, so that release can be triggered timely.

Reply:
@Abhishek-kumar09 I've confirmed, you'd be needing both of them, the cas-type name is `localpv-hostpath` but the Dynamic-LocalPV-provisioner adds the label `openebs.io/cas-type=local-hostpath`(something which would be fixed at a later stage after checking if this change doesn't break things like upgrade.